tests/queries/0_stateless/02912_ingestion_mv_deduplication.sql

```diff
@@ -48,7 +48,7 @@ SELECT '-- Original issue with deduplicate_blocks_in_dependent_materialized_view
     This is the unexpected behavior due to setting max_insert_delayed_streams_for_parallel_write > 1.
 
     This unexpected behavior was present since version 21.9 or earlier but due to this PR https://github.com/ClickHouse/ClickHouse/pull/34780
-    when max_insert_delayed_streams_for_parallel_write setting it to 1 by default the issue was mitigated.
+    when max_insert_delayed_streams_for_parallel_write gets disabled by default the issue was mitigated.
```

Comment:
@alexey-milovidov I had some doubts about merging this test in #56621 since the behavior of this part of the test is unexpected and should be fixed.

Do you think it's a good idea to keep it and leave it here as this and remove it as part of fixing the issue described in https://github.com/ClickHouse/ClickHouse/issues/56642? Or better to remove it and leave tests with only the desired behavior?